public/index.html

```diff
@@ -14,9 +14,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>To do</title>
-    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Material+Icons|Roboto|Montserrat:300,400,500,700" rel="stylesheet">
```

Comment:
We should not include external resources without integrity marks (security reasons). For google fonts that won't work. I think we should bundle fonts, but let's make that a separate PR.